src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/dialog/clone/CloneRadDialog.java

```diff
@@ -109,12 +109,6 @@ private void setActiveProfile() {
         });
     }
 
-    @Override
```

Comment:
Not really related to this change, but can we please change the _Open in browser_ hint color? Red is related to errors and while it gets the user's attention, it does so for all the wrong reasons :)